src/neynarAPI/neynarV2API/swagger/spec.yaml

```diff
@@ -0,0 +1,977 @@
+          type: array
+          items:
+            $ref: "#/components/schemas/Address"
+        activeStatus:
+          $ref: "#/components/schemas/ActiveStatus"
+    CastEmbeds:
+      oneOf:
+        - $ref: "#/components/schemas/EmbedUrl"
+        - $ref: "#/components/schemas/EmbedCastId"
+    Cast:
+      type: object
+      required:
+        - hash
+        - parent_hash
+        - parent_url
+        - parent_author
+        - author
+        - text
+        - timestamp
+        - embeds
+      properties:
+        hash:
+          type: string
+        parent_hash:
+          type: string
+          nullable: true
+        parent_url:
+          type: string
+          nullable: true
+        parent_author:
+          type: object
+          allOf:
+            - $ref: "#/components/schemas/Fid"
+            - required:
+                - fid
+              properties:
+                fid:
+                  type: string
+                  nullable: true
+        author:
+          $ref: "#/components/schemas/User"
+          # oneOf:
+          #   - $ref: '#/components/schemas/User'
+          #   - type: object
+          #     required:
+          #       - fid
+          #     properties:
+          #       fid:
+          #         type: string
+        text:
+          type: string
+        timestamp:
+          $ref: "#/components/schemas/Timestamp"
+        embeds:
+          type: array
+          items:
+            $ref: "#/components/schemas/CastEmbeds"
+        type:
+          $ref: "#/components/schemas/CastNotificationType"
+    ProfileUrl:
+      type: object
+      required:
+        - pfp
+      properties:
+        pfp:
+          type: object
+          required:
+            - url
+          properties:
+            url:
+              type: string
+    SearchedUser:
+      allOf:
+        - $ref: "#/components/schemas/User"
+        - $ref: "#/components/schemas/ProfileUrl"
+    RegisterSignerKeyReqBody:
+      type: object
+      required:
+        - signer_uuid
+        - signature
+        - app_fid
+        - deadline
+      properties:
+        signer_uuid:
+          type: string
+          description: UUID of the signer
+        signature:
+          type: string
+          description: Signature generated by the custody address of the app. Signed data includes app_fid, deadline, signer’s public key
+        app_fid:
+          type: integer
+          description: Application FID
+        deadline:
+          type: integer
+          description: unix timestamp in seconds that controls how long the signed key request is valid for. (24 hours from now is recommended)
+    PostCastReqBody:
+      type: object
+      required:
+        - signer_uuid
+        - text
+      properties:
+        signer_uuid:
+          type: string
+        text:
+          type: string
+        embeds:
+          $ref: "#/components/schemas/CastEmbeds"
+        parent:
+          $ref: "#/components/schemas/Address"
```

Comment:
Is that right? What about replying to other casts, or replying to a URL?

Reply:
It says address but that's just the name and description. It only checks to make sure it's a string.

```
  schemas:
    Address:
      type: string
      description: Ethereum address
```